packages/compiler/core/nodeHost.ts

```diff
@@ -36,4 +36,16 @@ export const NodeHost: CompilerHost = {
   },
   logSink: createConsoleSink(),
   mkdirp: (path: string) => mkdirp(path),
+  fileURLToPath,
+  pathToFileURL(path: string) {
+    return pathToFileURL(path).href;
+  },
 };
+
+export const cadlVersion = getVersion();
+
+function getVersion(): string {
```

Comment:
Might be able to change that to use `import` on the package.json to have `vite` resolve the value on build.

Reply:
I seem to recall we had it this way some time back but changed it to this for reasons. @nguerrera  do you remember anything like this?

Reply:
I also think this is fine for now. We can fix this later if we need to!

Reply:
I don't recall having it that way.